src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/MessageTrailerTest.java

```diff
@@ -13,21 +12,11 @@
     private static final int NUMBER_OF_SEGMENTS = 18;
     private static final long SEQUENCE_NUMBER = 3L;
 
-    @Test
-    public void testValidMessageHeader() {
-        MessageTrailer messageTrailer = new MessageTrailer(NUMBER_OF_SEGMENTS);
-        messageTrailer.setSequenceNumber(SEQUENCE_NUMBER);
-
-        String edifact = messageTrailer.toEdifact();
-
-        assertEquals("UNT+18+00000003'", edifact);
-    }
-
     @Test
     public void testValidationStatefulNonSequenceNumber() {
```

Comment:
```suggestion
   void testValidationStatefulNonSequenceNumber() {
```

Reply:
What will this change?

Reply:
First of all, we will stay consistent with other tests we have. Secondly, the JUnit 5 test classes and methods don't need to be public. They can be restricted to default visibility.